sunpy/map/maputils.py

```diff
@@ -81,6 +81,14 @@ def map_edges(smap):
 
 
 @u.quantity_input
+def _is_coordinates_frame_helioprojective(coordinates):
```

Comment:
This feels a tad long as a function name.

Reply:
What about `_is_coordinate_helioprojective(coordinates)` ?

Reply:
There is an PEP8 issue at [Line 244:101](https://github.com/sunpy/sunpy/blob/ec1789c56c04c3db7bfd1b85b5c01766c58568eb/sunpy/map/maputils.py#L244): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (101 > 100 characters).

So shall I make a new variable named `coordinate_angular_distance` whose value is `np.sqrt(coordinates.Tx ** 2 + coordinates.Ty ** 2)` 
and replace `return np.sqrt(coordinates.Tx ** 2 + coordinates.Ty ** 2) < solar_angular_radius(coordinates)` to 
`return coordinate_angular_distance < solar_angular_radius(coordinates)`

Reply:
I would ignore that.

Reply:
any sugggestions?

Reply:
> What about `_is_coordinate_helioprojective(coordinates)` ?

This works.